parser/pt_parser.go

```diff
@@ -0,0 +1,69 @@
+// PT parser pasres the Paris TraceRoute .web100 files, and output a ValueSaver
+package parser
+
+import (
+	"bufio"
+	"fmt"
+	//"io"
+	"cloud.google.com/go/bigquery"
+	"os"
+	"strings"
+	//"golang.org/x/net/context"
+	//"google.golang.org/api/iterator"
+)
+
+// PT implements the ValueSaver interface.
+type PT struct {
+	test_id  string
+	project  int
+	log_time int
+	connection_spec
+}
+
+type connection_spec struct {
+	server_ip      string
+	server_af      int
+	client_ip      string
+	client_af      int
+	data_direction int
+}
+
+// Save implements the ValueSaver interface.
+func (i *PT) Save() (map[string]bigquery.Value, string, error) {
```

Comment:
FYI, though I think this should work, I ended up having to avoid pointer receiver to get a similar implementation to work with bigquery.